chanzuckerberg/idseq-web
# [Report Page] Only show delete button if the sample is deletable

# Description

* Deletable if user is admin, there are not pipeline runs OR all pipeline runs have failed
* Ensure that the `Sample.deletable?` method return a boolean

# Notes

* This change has no effects on the legacy page

# Tests

* Check that admin sees the delete button for sample (currently only while the report is not ready)
* Check that normal user only see delete button for sample:
	- the user owns AND
	- have no successful pipeline runs
